pkg/sync/sync.go

```diff
@@ -56,6 +56,8 @@ func AssetNames(cs *api.OpenShiftManagedCluster) ([]string, error) {
 
 func Asset(cs *api.OpenShiftManagedCluster, name string) ([]byte, error) {
 	switch cs.Config.PluginVersion {
+	case "v13.0", "v13.1":
+		return v13.Asset(name)
```

Comment:
This needed to be added back, looks like it was taken out by accident: https://github.com/openshift/openshift-azure/commit/3ceae6a9bd8c8368880bfb822db6109fe3ab4224#diff-30a1d1dc9ee97fc02edf2488cbf6ff6cL60